wiki/models/urlpath.py

```diff
@@ -41,7 +41,10 @@ class URLPath(MPTTModel):
     article = models.ForeignKey(Article, on_delete=models.CASCADE, editable=False,
                                 verbose_name=_(u'Cache lookup value for articles'))
     
-    slug = models.SlugField(verbose_name=_(u'slug'), null=True, blank=True)
+    SLUG_MAX_LENGTH = 255
```

Comment:
Nit: Can you add a comment here mentioning that, while this field is capped at 255, in practice that limit is much smaller (65) due to https://github.com/edx/edx-platform/blob/98b20d1fc4ac1a8728572a55052279856ff50ec3/common/static/common/js/components/utils/view_utils.js#L16-L17?